### PR TITLE
MBS-8909: Add IMDb and VGMdb favicons to entity sidebar

### PR DESCRIPTION
**Pull request by caroline-gschwend**

### CODE REVIEW

File: root/static/styles/favicons.less [hunk 1]
```diff
@@ -34,6 +34,7 @@
 .favicon("genius");
 .favicon("googleplus", 32);
 .favicon("ibdb");
+.favicon("IMDb");
```
Comment (Freso) [inline root/static/styles/favicons.less]: This shouldn't have the `, 32` like googleplus and instagram etc.? (I don't know what that value is, I just noted that the imdb.png is called -32 while the vmdb one is -16…)

Comment (Freso) [inline root/static/styles/favicons.less]: Also, all the others are all lowercase, so this and VGMdb should probably be as well.

**Comment by gentlecat** [comment]
@brainzbot, add to whitelist.

File: root/static/styles/favicons.less (hunk 1 above)
Comment (caroline-gschwend) [inline root/static/styles/favicons.less]: @Freso , what about "iTunes" and "SMDB"? I'm confused on the capitalizations here.

Comment (Freso) [inline root/static/styles/favicons.less]: I only looked at the ones in the diff. :) I would go with the more prevalent style, which, based on the diffs, seem to be all lower vase.

Comment (caroline-gschwend) [inline root/static/styles/favicons.less]: @Freso, do you want me to change all of them?

File: root/static/styles/favicons.less [hunk 2]
```diff
@@ -74,6 +75,7 @@
 .favicon("tunearch");
 .favicon("twitter");
 .favicon("viaf", 32);
+.favicon("vgmdb");
```
Comment (mwiencek) [inline root/static/styles/favicons.less]: nitpick, but that should go above viaf

**Comment by mwiencek** [comment]
To make the new icons get used, you also have to modify `FAVICON_CLASSES` in lib/MusicBrainz/Server/Entity/URL/Sidebar.pm. Here's a helpful example: https://bitbucket.org/metabrainz/musicbrainz-server/pull-requests/1646/mbs-8560-add-google-play-to-the-sidebar/diff

**Comment by mwiencek** [comment]
Don't worry about the test failure, it's random and unrelated (gotta figure out how to fix that soon).

**Comment by mwiencek** [comment]
Thanks! 👍

**Pull request merged**